AprilSylph/XKit-Rewritten
# New Tweak: Subtle following/mutuals indicators

<details>
<summary>outdated</summary>
<img width="401" src="https://github.com/AprilSylph/XKit-Rewritten/assets/8336245/10700d85-28fc-45f3-a873-be450c72663c">

Just for funsies for now. No idea how to get this to actually work properly.
</details>

### Description


Adds a tweak that, rather than showing or completely hiding the following/mutuals icons in activity, makes them only appear as icons until the user hovers over them.

As this is a CSS vanity project, this has some unnecessary complexity to implement a smooth show-the-label animation. Specifically, putting a transition on the width property requires a hardcoded final width (grabbed via `getBoundingClientRect`) rather than just setting `width: unset`. It would be quite easy to remove this and make the label instantly appear; mention me if that's desired.

### Testing steps

Enable the tweak and observe its functionality on the activity page and activity modal.
